### PR TITLE
908992-Added opening and saving Word document in Google Drive cloud storage samples

### PR DESCRIPTION
**Pull request by DharanitharanA**
Added opening and saving Word document in Google Drive cloud storage samples

### CODE REVIEW

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
+                    GoogleClientSecrets.Load(stream1).Secrets,
+                    Scopes,
+                    "users",
+                    CancellationToken.None,
+                    new FileDataStore(credPath, true)).Result;
+            }
+
+            // Step 2: Create Drive API service
+            var service = new DriveService(new BaseClientService.Initializer()
```
Comment (LokeshBaskar) [inline Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs:28]: Add pull stop at the end of the comment.

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+            // Step 4: Download the Word document from Google Drive
+            var request = service.Files.Get(fileId);
+            var stream = new MemoryStream();
+            request.Download(stream);
+
+            // Step 5: Save the Word document locally
+            using (FileStream fileStream = new FileStream("Output.docx", FileMode.Create, FileAccess.Write))
+            {
+                stream.WriteTo(fileStream);
```
Comment (LokeshBaskar) [inline Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs:45]: Dispose the stream

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+            textRange = paragraph.AppendText("Price: $3,578.27\r") as WTextRange;
+            textRange.CharacterFormat.FontSize = 12f;
+            textRange.CharacterFormat.FontName = "Times New Roman";
+
+            //Appends paragraph
+            section.AddParagraph();
+
+            //Saves the Word document to MemoryStream
+            MemoryStream stream = new MemoryStream();
+            document.Save(stream, FormatType.Docx);
```
Comment (LokeshBaskar) [inline Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs:242]: Dispose document

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+            // Create metadata for the file to be uploaded
+            var fileMetadata = new File()
+            {
+                Name = "Output.docx", // Name of the file in Google Drive
+                MimeType = "application/msword",
+            };
+            FilesResource.CreateMediaUpload request;
+            // Create a memory stream from the Word document
+            using (var fs = new MemoryStream(stream.ToArray()))
+            {
```
Comment (LokeshBaskar) [inline Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs:276]: Dispose the stream

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs (hunk 1 above)
Comment (DharanitharanA) [reply Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs:28]: Added properly.

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs (hunk 2 above)
Comment (DharanitharanA) [reply Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs:45]: Handled properly.

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs (hunk 3 above)
Comment (DharanitharanA) [reply Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs:241]: Disposed properly.

File: Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs (hunk 4 above)
Comment (DharanitharanA) [reply Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Save-Word-document/Save-Word-document/Program.cs:275]: Disposed properly.

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Review by Ramaraj-Marimuthu — APPROVED** [review]

**Pull request merged**